packages/node/src/node.ts

```diff
@@ -44,12 +49,22 @@ export class Node {
       // account-address-based indexing
       `${nodeConfig.STORE_KEY_PREFIX}/${this.signer.address}`
     );
+    this.instructionExecutor = new InstructionExecutor({
+      // TODO: Pass in NetworkContext into Node constructor
```

Comment:
This should be updated now that `NetworkContext` is being passed in.

Reply:
Done